src/pages/ReimbursementAccount/ValidationStep.js

```diff
@@ -51,179 +51,169 @@ const defaultProps = {
     },
 };
 
-class ValidationStep extends React.Component {
-    constructor(props) {
-        super(props);
+/**
+ * Filter input for validation amount
+ * Anything that isn't a number is returned as an empty string
+ * Any dollar amount (e.g. 1.12) will be returned as 112
+ *
+ * @param {String} amount field input
+ * @returns {String}
+ */
+const filterInput = (amount) => {
```

Comment:
Why are we moving this function alone outside of the functional component?

Reply:
@BartoszGrajdek

Reply:
Hi, that's because this function is not dependent on the component - meaning no state passed, refs etc. It just improves the readability of the component overall making it easier to understand. Let me know what you think @abdulrahuman5196 😄